npm/ipsm/ipsm_test.go

```diff
@@ -568,28 +556,35 @@ func TestRun(t *testing.T) {
 
 func TestRunError(t *testing.T) {
 	setname := "test-set"
+	setname2 := "test-set2"
 
 	var calls = []testutils.TestCmd{
 		{Cmd: []string{"ipset", "-N", "-exist", util.GetHashedName(setname), "nethash"}, Stdout: "test failure", ExitCode: 2},
 	}
 
-	fexec, fcmd := testutils.GetFakeExecWithScripts(calls)
-
+	fexec := testutils.GetFakeExecWithScripts(calls)
 	ipsMgr := NewIpsetManager(fexec)
+	defer testingutils.VerifyCalls(t, fexec, calls)
```

Comment:
This testset has failed in azure pipeline. 
I think the reason is below code.
```go
# VerifyCalls in utils.go.
require.Nil(t, err) 
```  
It seems using "require.NotNil(t, err)" is right for this testset. 
Please check it.

Reply:
since we're importing testify anyway, the convenience methods read better than checking nil:
`assert.Error(t, err)` or `assert.NoError(t, err)`, whichever it should be

Reply:
I typically like `require.NoError` for failfast behavior, I used it elsewhere in this PR so I'll update for consistency